### DISCUSSION TITLE
Using 'with-poll' macOS feature with CIDER

### DISCUSSION THREAD

**Discussion by jmckitrick** [clojure-emacs/cider#3740] [General]
I’m trying out the ‘with-poll’ feature of emacs on macos, and it seems to be a drop-in replacement for the current design. However, when I try to run CIDER and connect to a running instance, nrepl connect times out. I don’t know the internals well enough to guess if this is an emacs and polling problem or a cider and nrepl problem. Or maybe just an adjustment of expectations on one side or the other. Any suggestions on where to start?

**Comment by vemv** [comment]
Some previous chat https://github.com/clojure-emacs/cider/issues/3455

**Comment by jmckitrick** [comment]
@vemv Ah, the joys of middle age. Sorry I forgot about that thread lol.
